Kernel/CMakeLists.txt

```diff
@@ -91,10 +87,13 @@ set(KERNEL_SOURCES
     Graphics/VirtIOGPU/Console.cpp
     Graphics/VirtIOGPU/GPU3DDevice.cpp
     Graphics/VirtIOGPU/GraphicsAdapter.cpp
-    IOWindow.cpp
-    Jail.cpp
-    JailManagement.cpp
     SanCov.cpp
+    Security/AddressSanitizer.cpp
+    Security/Credentials.cpp
+    Security/Jail.cpp
+    Security/JailManagement.cpp
+    Security/Random.cpp
```

Comment:
should this file be named `SecureRandom` instead? (`Random` itself is too generic name to put it under security, imho; to some extent both could maybe even go to `Devices`!?).

Reply:
Feels redundant to me to call it `SecureRandom` in `Security` folder. Also, one of the key points of using randomness is for security so for me it feels very related (the concept, the actual impl should go in another directory ofc).

Reply:
well… i wouldnʼt accept the argument on key points—applied elsewhere (to other elements) could lead to more obvious misleads.

in my opinion randomness as such should be treated just as a phenomenon. iʼd put its usages aside (i.e. you donʼt know how somebody will use this generic concept; now youʼre telling, for your own convenience – which is not a bad thing – that ones key point will be security, even if it wonʼt be in some cases). no big deal with that, probably just another move in the future (if necessary). 👍